src/Makefile

```diff
@@ -110,23 +110,28 @@ _SOURCES=\
 	generic_chunk.c \
 	gorilla.c \
 	indexer.c \
+	endianconv.c \
```

Comment:
we assume that this is part of redis, by including this we might override the redis symbols.

Reply:
thank you Danni. fixed in the last commit